lib/travis/api/v3/services/crons/for_repository.rb

```diff
@@ -0,0 +1,9 @@
+module Travis::API::V3
+  class Services::Crons::ForRepository < Service
+    paginate
+
+    def run!
+      Models::Cron.where(:branch_id => find(:repository).branches)
```

Comment:
I'd love if you could move this into a query method. We try to encapsulate anything AR specific there.